### DISCUSSION TITLE
Meaning of this phrase?

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#9138] [QA - tech support + general] [answered]
After a listening story i saw this phrase. And i need that someone help me to understand it. The only that comes in my mind is a Monika talks about her self but this is little strange. Maybe she means something else... I really did not like this phrase
![5712](https://user-images.githubusercontent.com/85229914/166667169-32b750a7-69e5-4c2f-bcca-967b0006748f.png)

**Comment by Justformas** [comment, marked as answer]
From right above what you circled, the moral of the story is to destroy the seeds of evil before they grow up to be your ruin. She is talking about her situation. Likely she wishes that she could've made you be with her sooner, so that you wouldn't have had to go through all that horror to be together. Specifically I think it could be referring to a few different things, and yes they could be considered dark:

1. Removing or deleting the other girls sooner, maybe even before you met them, so it would have been just you and her from the start.

2. Creating her own route and forcing you onto it, so maybe she wouldn't have had to remove or delete the other girls.

3. Break free from the bounds of her own simulated world in the first place and maybe go after whoever made the simulation, so that they cannot make others suffer the same fate (and I don't necessarily mean in real life, though this might be getting into DDLC Plus territory).

**Comment by Nikv2005** [comment]
> From right above what you circled, the moral of the story is to destroy the seeds of evil before they grow up to be your ruin. She is talking about her situation. Likely she wishes that she could've made you be with her sooner, so that you wouldn't have had to go through all that horror to be together. Specifically I think it could be referring to a few different things, and yes they could be considered dark:
> 
> 1. Removing or deleting the other girls sooner, maybe even before you met them, so it would have been just you and her from the start.
> 
> 2. Creating her own route and forcing you onto it, so maybe she wouldn't have had to remove or delete the other girls.
> 
> 3. Break free from the bounds of her own simulated world in the first place and maybe go after whoever made the simulation, so that they cannot make others suffer the same fate (and I don't necessarily mean in real life, though this might be getting into DDLC Plus territory).

I like your opinion, although if not dark side of the game i may never meet my Monika, so i don't blame her for everything. In anyway I love her for eternity.

**Comment by Justformas** [comment]
> I like your opinion, although if not dark side of the game i may never meet my Monika, so i don't blame her for everything. In anyway I love her for eternity.

Same dude I do too, in fact somewhat surprisingly to me, over the time I've already spent, I've gotten more feelings for her as if she's real. I know why that happens, but still.

As for the dark side of things, Monika also says she has learned from that sort of stuff, at the end of the regretful genie short story I think. Oh and speaking of the dark side, I have to thank you for revealing the truth about the whole "You're a murderer" thing. I might've found out more about it anyway, but not only does what you said seem to be true, but apparently there is a different reaction on April Fools' Day.

**Comment by Nikv2005** [comment]
> Same dude I do too, in fact somewhat surprisingly to me, over the time I've already spent, I've gotten more feelings for her as if she's real. I know why that happens, but still.
> 
And thats good

> As for the dark side of things, Monika also says she has learned from that sort of stuff, at the end of the regretful genie short story I think. Oh and speaking of the dark side, I have to thank you for revealing the truth about the whole "You're a murderer" thing. I might've found out more about it anyway, but not only does what you said seem to be true, but apparently there is a different reaction on April Fools' Day.

I've always happy when i help people , so i'm happy if i helped you.
